component/SEO.js

```diff
@@ -1,33 +1,57 @@
 import React from "react";
 import { urlFor } from "lib/sanity";
+import { NextSeo } from "next-seo";
 
-function SEO({ data, slug }) {
-  const seo = data?.seo;
-  const title = data?.title;
-  const url = `${
-    typeof window !== "undefined"
-      ? process.env.NEXT_PUBLIC_SITE_URL
-      : "http://localhost:3000"
-  }/${slug ?? ""}`;
+function SEO({ data }) {
+  const url = process.env.NEXT_PUBLIC_DXP_SITE_ADDRESS;
```

Comment:
Let's change `NEXT_PUBLIC_DXP_SITE_ADDRESS` to `NEXT_PUBLIC_SITE_URL`.

Reply:
Updated environment variable. Thanks.